pages/help.py

```diff
@@ -489,13 +488,12 @@ def get_content(self):
 aware of and are working on.
 """)),
                     FaqItem("kaspersky", _("I have Kaspersky anti-virus, and Freenet doesn't install, or shows \"Download/upload queue database corrupted!\""), _("""
-                    Kaspersky can be a problem with Freenet. See [here](
-https://wiki.freenetproject.org/Installing/Windows#.27Download
-.2Fupload_queue_database_corrupted.21.27_.28When_using_Kaspersky_on_Windows_7
-.29). We recommend you turn off Kaspersky during install and during node
-startup, and exclude the directory you installed Freenet in (most likely
-C:\Program Files\Freenet or C:\Program Files (x86)\Freenet).
-""")),
+Kaspersky can be a problem with Freenet. See [here][url_kaspersky].
+We recommend you turn off Kaspersky during install and during node startup, and exclude the directory you installed Freenet in (most likely {freenet_install_dir_1} or {freenet_install_dir_2}).
+""").format(freenet_install_dir_1="C:\Program Files\Freenet",
+            freenet_install_dir_2="C:\Program Files (x86)\Freenet") + "\n\n" + """
```

Comment:
The firectories are indeed localized, however [for most languages][0] it's plain "Program Files". Given the text mentions "most likely", there's no point in translating this. Then again users will probably know what "Program Files" means, but may not understand the reverse translation if they happen to be using an international Windows installation.

[0]: https://en.wikipedia.org/wiki/Program_Files